addons/purchase_edi_ubl_bis3/models/purchase_order.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def _get_edi_builders(self, edi_builders):
```

Comment:
No need to pass edi_builders as arguments, you could just return the list and an empty list if there is no builder